persistence/provider/sql/sqlite/schema.go

```diff
@@ -110,12 +110,21 @@ func createEventStoreSchema(ctx context.Context, db *sql.DB) {
 	sqlx.Exec(
 		ctx,
 		db,
-		`CREATE INDEX repository_query ON event (
+		`CREATE INDEX by_type ON event (
 			source_app_key,
 			portable_name,
-			offset,
+			"offset"
```

Comment:
🐼 

```suggestion
			offset
```

These don't need to be quoted in SQLite, for whatever reason. Just doing this to mimimize the diff/churn.